widgets/executionLogs/src/widget.tsx

```diff
@@ -51,6 +54,17 @@ Stage.defineWidget<ExecutionLogsParams, ExecutionLogsData, ExecutionLogsConfigur
             return <Message>{translate('noExecutionSelectedMessage')}</Message>;
         }
 
-        return <LogsTable data={data} />;
+        const items = [...data.items].reverse();
+        const moreItemsAvailable = data.metadata.pagination.size >= data.metadata.pagination.total;
```

Comment:
When `size` equals `total`, doesn't it mean that there are no more items available to fetch? 🤔
 
Shouldn't this condition be stated with just a ">" sign?

Example:
```suggestion
        const moreItemsAvailable = data.metadata.pagination.size > data.metadata.pagination.total;
```

Reply:
It's intentional. When `size` (number of items to fetch) equals to `total` (number of all available items) we are sure that all items have already been fetched.

Reply:
Okay, got it
I had to analyze the whole use case to correct my confusion, descriptions related to `size` and `total` have helped a lot! :)

Thanks for the clarification, in that case everything is looking good 💪